src/ocm/components/clusterWizard/wizardTransition.ts

```diff
@@ -37,18 +46,48 @@ export const getClusterWizardFirstStep = (
     }
     return 'static-ip-network-wide-configurations';
   }
+  const step = getStepIdByHostValidations(hosts);
   switch (state) {
     case 'ready':
       return 'review';
     case 'pending-for-input':
     case 'adding-hosts':
     case 'insufficient':
-      return 'host-discovery';
+      return step ? step : 'host-discovery';
```

Comment:
We probably shouldn't let `step` be before `host-discovery` as no errors can be fixed in those early steps.
Either use a different map to validate the function, or add some logic here to put it back to host-discovery.